ci_tools/coverage_analysis_tools.py

```diff
@@ -259,10 +259,10 @@ def get_json_summary(untested, content_lines, existing_comments, diff):
             else:
                 end_line = line_indices[-1]
             last_valid_line = start_line
-            last_valid_line_idx = next(i for i in diff[f]['addition'] if i == last_valid_line)
-            for i in diff[f]['addition'][last_valid_line_idx:]:
-                if i-last_valid_line < 6:
-                    last_valid_line = i
+            last_valid_line_idx = next(k for k,l in enumerate(diff[f]['addition']) if l == last_valid_line)
```

Comment:
Are we sure that `last_valid_line` will be present in the dictionary?

Reply:
Yes because `untested` is the output of:
https://github.com/pyccel/pyccel/blob/28f555d3ccd91918e0137ea8d98a9f22a0942797/ci_tools/coverage_analysis_tools.py#L47-L75